### PR TITLE
initial commit for migration article

### PR DESCRIPTION
**Pull request by SebastianKuehnau**
design needs to be check, it didn't work on my machine.

### CODE REVIEW

**Comment by paulroemer** [comment]
We do not use topic folders anymore. So remove the `migration` folder and just put it into `tutorials`

**Comment by paulroemer** [comment]
Get your UUID from https://preview.vaadin.com/company/team and add it as a property `author_id`.

File: tutorials/migration/v7-v10+-migration/content.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
+= Practical steps to migrate a simple Vaadin 7 application to Vaadin Platform
+
+:title: Practical steps to migrate a simple Vaadin 7 application to Vaadin Platform
+:authors: sebastian
+:type: text
+:topic: frontend
```
Comment (paulroemer) [inline tutorials/migration/v7-v10+-migration/content.adoc]: `frontend` is not a valid topic. Check topics.json.

**Comment by marcushellberg** [comment]
Sorry @SebastianKuehnau, didn't notice this until now. Is this ready for review and merge from your side?

**Pull request merged**